ordered.go

```diff
@@ -3,8 +3,8 @@ package lang
 // Ordered is an interface that collects all primitives that can be ordered. This is typically helpful when creating
 // Go 1.18 generic definitions:
 //
-//     func compare[T lang.Ordered](a, b T) int {
-//     }
+//	func compare[T lang.Ordered](a, b T) int {
+//	}
```

Comment:
Does this really mean the new linter cares about indentation of commented "code"(-like) lines? Weird.

Reply:
I think `golangci-lint` and `gofmt` always cared. In CI, `golangci-lint` was reporting two `Error: File is not `gofmt`-ed with `-s` (gofmt)`, but it did not report which files, while on my local machine `golangci-lint` reported no non-formatted files. When I ran `gofmt -s -w .`, locally in the project root directory, it formatted these two files.

Reply:
I object to formatters messing with the contents of my block comments, but, then again, I generally don't fight (too much) with linters....  🦐

Reply:
I object to a linter thinking that anything in comments is "code": but, yeah, arguing with a linter is generally unproductive. (Unless one wants to get into arguing with the linter's developer, at least.)